host_vars/hostname.example

```diff
@@ -63,24 +68,34 @@ local_mounts: []
 #     src: /dev/sdc1
 #     fstype: ext3
 
-# If you want to use letsencrypt to generate SSL key/cert. Otherwise you MUST
-# put your key and cert into the security directory as security/ssl.crt and 
-# security/ssl.key
+# SSL using letsencrypt (optional - use letsencrypt default: false)
+# If using letsencrypt to generate SSL key/cert, set `use_letsencrypt` to `true`
+# Otherwise if not using letsencrypt for SSL, you MUST put your key and cert into the security directory as
+# `security/ssl.crt` and `security/ssl.key`
 use_letsencrypt: false
 letsencrypt_email: ''
 
-# If you want users to have /public_html/username directories set this to true
+# If users should have `/public_html/username` directories, set this to `true` (Optional)
 nginx_public_html: false
 
-# Optionally set this to a list of the GitHub usernames whose public
-# GitHub SSH keys will be installed to allow then to ssh to the server
-# as root.
+# List of the GitHub usernames who will receive root access via ssh
+# Public GitHub SSH keys will be installed to allow the user to ssh into the server as root.
 github_usernames: ['instructor', 'grader']
 
-# Optionally cull idle single user servers
+# Oauth (Optional - use oauth default: false)
+# Change `use_oauth` to `true` and set the other values appropriately
+# to enable GitHub OAuth.
+# If left as default `false`, PAM authentication will be used.
+use_oauth: false
+oauth_callback_url: https://mydomain.org/hub/oauth_callback
+oauth_client_id: ''
+oauth_client_secret: ''
+
+# Cull idle servers (Optional default: false)
+# Enable or disable cull idle single user servers
 use_cull_idle_servers: false
 # The username for culling idle servers. MUST be a user of JupyterHub,
-# so root won't work unless root can access JupyterHub
+# as root won't work unless root can access JupyterHub
```

Comment:
'As' is a bit less accurate than 'so', since it's explaining something that's a consequence of the previous clause. Perhaps "For example, root won't work..."